crates/holochain_sqlite/src/db.rs

```diff
@@ -270,6 +270,22 @@ impl<Kind: DbKindT> DbRead<Kind> {
             Err(e) => Err(DatabaseError::Timeout(e)),
         }
     }
+
+    #[cfg(any(test, feature = "test_utils"))]
+    pub fn test_read<R, F>(&self, f: F) -> R
+    where
+        F: FnOnce(Transaction) -> R + Send + 'static,
+        R: Send + 'static,
+    {
+        // Necessary to first tell Tokio that we are intentionally blocking on a possibly non-blocking thread
+        tokio::task::block_in_place(|| {
+            tokio::runtime::Handle::current().block_on(async {
```

Comment:
BTW, this is how wasm calls back into holochain async functions. Using [the utility function here](https://github.com/holochain/holochain/blob/43f0aa760bbba1afed8415d66c3f901a77d4655e/crates/holochain_util/src/tokio_helper.rs#L37), which we could use here too.

Reply:
Good point, done.